### PR TITLE
Copy to clipboard directive

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -94,10 +94,18 @@ import { TxConfirmNotificationProvider } from '../providers/tx-confirm-notificat
...
 
+/* Directives */
+
+import { CopyToClipboard } from '../directives/copy-to-clipboard/copy-to-clipboard';
+
 export function createTranslateLoader(http: HttpClient) {
   return new TranslatePoHttpLoader(http, 'assets/i18n', '.po');
 }
 
+let declarations: any = [
```
Comment (gabrielbazan7) [inline src/app/app.module.ts]: Baybe this array should be name as "directives" ?

Comment (cmgustavo) [reply src/app/app.module.ts]: Ok, I agree.

Comment (cmgustavo) [reply src/app/app.module.ts]: Updated. Thanks!

**Comment by gabrielbazan7** [comment]
ACK

**Pull request merged**